arches/app/models/migrations/11848_alter_spatialview_geometrynode.py

```diff
@@ -0,0 +1,156 @@
+import django.db.models.deletion
+from django.db import migrations, models
+import pgtrigger
+
+
+class Migration(migrations.Migration):
+    drop_legacy_trigger = "drop trigger if exists __arches_trg_update_spatial_views on spatial_views; drop function if exists __arches_trg_fnc_update_spatial_views;"
+
+    create_legacy_trigger = """
```

Comment:
Just wondering if we want to start externalizing long-lasting SQL or if the plan is to keep it embedded in the .py migration file. My preference would be to start pulling it out for readability.

Reply:
In this case, this sql is not particularly long lasting - this is just a rollback and a drop.  Past this point, the long lasting sql gets managed in the sql file.